src/main/java/frc/robot/commands/autoCommands/waitCommand.java

```diff
@@ -0,0 +1,31 @@
+package frc.robot.commands.autoCommands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class waitCommand extends Command {
```

Comment:
No need to duplicate this as an existing WaitCommand already exists: https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj2/command/WaitCommand.html

I'm going to add some other comments to this command just to help improve your coding ability, but the correct answer is to use the above class instead of making our own.